vite.config.mts

```diff
@@ -0,0 +1,81 @@
+/**
+ * Copyright (c) 2024, RTE (http://www.rte-france.com)
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+
+import react from '@vitejs/plugin-react';
+import type { PluginOption } from 'vite';
+import { defineConfig } from 'vite';
+import * as path from 'path';
+import { resolve } from 'path';
+import eslint from 'vite-plugin-eslint';
+import dts from 'vite-plugin-dts';
+import * as fs from 'fs/promises';
+import * as url from 'url';
+import { createRequire } from 'node:module';
+import svgr from 'vite-plugin-svgr';
+import { externalizeDeps } from 'vite-plugin-externalize-deps';
+
+export default defineConfig({
+    plugins: [
+        react(),
+        eslint(),
+        dts(),
+        svgr({ include: '**/*.svg' }), // default is { include: "**/*.svg?react" }
+        reactVirtualized(),
+        externalizeDeps({
+            include: [/^react-is(?:\/.*)?$/, /^@mui\/styled-engine(?:\/.*)?$/],
+        }),
+    ],
+
+    build: {
```

Comment:
Have found how to keep file files compiled separately in bundle:
```suggestion
    build: {
        rollupOptions: {
            output: {
                preserveModules: true,
                entryFileNames: '[name].js',
            },
        },
```

Result: 
![image](https://github.com/gridsuite/commons-ui/assets/135599584/229026ff-30cc-4ab2-b1b6-a12cabde607d)